### ISSUE TITLE
AstroHackWeek 2017 start-up

### ISSUE THREAD

**Issue by drphilmarshall** [AstroHackWeek/AstroHackWeek2017#1]
@jakevdp Here's what we said needed doing sooner rather than later (this fall?)
- [x] Confirm that UW can host
- [x] Find a main venue, and its dates of availability next August/September
- [x] Start looking into possible local sponsors, to fund travel grants for participants
- [x] Choose exact dates

Feel free to spin these out into separate sub-issues as required! :-)

**Comment by jakevdp** [comment]
Great! One thing that needs to be decided basically immediately is the space.

In 2014 we used the library's Active Learning Classroom (ALC). Another option would be the eScience studio. There are a few advantages and disadvantages of each
- The ALC technology is superior to anything we have in in eScience
- The ALC costs ~$2000 for the week; eScience would be free
- eScience does not have a room large enough to gather everyone for talks, though we can use the seminar room one floor below (This is what we did for PyAstro this past spring)
- The ALC is much less flexible regarding food – food and beverage is not allowed inside, but we can do coffee/lunch in the cafe space just outside the room. In eScience we can basically do whatever we want.
- The ALC is only available 9-4 each day, without flexibility. In 2014, we paid an overtime fee to extend that to 5. In eScience, we can have 24-hour access.
- For the ALC, we need to choose the week and reserve the space basically **right now** or someone else might do it before us. For eScience, we can be more flexible in our decision process.

Let me know your thoughts, particularly people who have been in both the ALC and the eScience space (@davidwhogg ?) We could plan to use the ALC during the day, then move to eScience for any evening gatherings.

**Comment by dhuppenkothen** [comment]
I like both spaces. My intuition is that the eScience space might be slightly too small for the entire group to hack, but I might be wrong. Jake, do you remember how many we were at PyAstro this year? 

I really, really liked the ALC space, I must admit. Both last year and this year those who were at all three Astro Hack Weeks agreed that it was by far the best space. Though 4pm is really early to finish during any given day, which isn't ideal. It does cost money, but if we start a bit earlier this year with organizing + fundraising, we might be able to get some sponsorship that might help us alleviate those costs. 

One thing we might try is a mix of the two? Something like three days in one space, two in the other? The ALS space seems superior for tutorials and break-outs because of the round tables and screens and whiteboard walls facilitating peer learning, compared to the seminar spaces at the eScience center, which might be better for more traditional lectures. The eScience center might work well for the actual hacking, if it's big enough. 

We worried before last year's Astro Hack Week that changing the space might be detrimental, but in the end, that turned out not to be true. I think both last year and this year the participants seemed to like the change of scenery (or were at least not deterred by it).

**Comment by jakevdp** [comment]
PyAstro was between 45-55, depending on the day.

> One thing we might try is a mix of the two? 

Neurohackweek is doing that this week. Mornings in the ALC, afternoons in the eScience space.

**Comment by dhuppenkothen** [comment]
So perhaps we can learn from them and see how that works out? Do we have time to wait for Thursday or Friday and ask Ariel what he thought about it?

**Comment by jakevdp** [comment]
Yes, we can definitely wait a week or so. I'll get Ariel's feedback on Monday.

One thing about afternoons in eScience though: one place where the ALC absolutely shined was in the daily wrap-up, where people could plug-in their computer at their table and project for everyone. Would be a shame to lose that.

**Comment by dhuppenkothen** [comment]
This is true, but if we have to be out of the ALC early every day, then that might not work either way. Did we really only stay until 5pm two years ago? I could have sworn it was 6pm ...

**Comment by jakevdp** [comment]
Oh, maybe it was 6pm? I'd have to check. There might be room for negotiation.

**Comment by jakevdp** [comment]
Looking at the 2014 schedule, it was 6pm. So that's probably still a possibility.

**Comment by drphilmarshall** [comment]
I would worry about committing to a space that was too small. If we can get
the alc until 6 again that would be ideal I think. If we can (later!) get
space at amazon (or somewhere!) on the weds, presumably that would reduce
our costs. What's the alc cancellation policy?

**Comment by davidwhogg** [comment]
We could also do ALC until 4, then move to eScience and do the end-of-day stand-up there. Or something more creative than that. I do LOVE the ALC, but we have also learned from the last two AHWs that changing venue is a good thing.

**Comment by jakevdp** [comment]
Hi folks, I just submitted a request to use the ALC the week of August 28-Sept 1st, 2017 (same week as this year's hack week). That doesn't set the dates in stone, but gets the conversation started with the folks who can book the room. If anyone has issues with those dates, please speak up!

**Comment by drphilmarshall** [comment]
Excellent! Thanks Jake. I'll start a separate issue on a possible day trip.

**Comment by jakevdp** [comment]
Hi all – I have a tentative hold on UW's Active Learning Classroom from August 28-September 1st, 9am-6pm each day. Including the 5-6pm overtime, the cost will be $565 per day (eScience can help defray some of this cost). Those times are hard open and close; there will be no access to the room before and after.

Food is not allowed in the room, nor are open beverages (we can arrange to provide lids for coffee, I believe).

I'm checking to see whether we can change our reservation if we choose to do an off-site day.

Let me know if all that sounds OK, and I'll sign the contract!

**Comment by dhuppenkothen** [comment]
What is our budget going to be? Is there going to be some Moore-Sloan money as in the last few years, or do we need to get active with fundraising for more than travel grants. If so (or perhaps more generally), we should probably start right _now_

**Comment by davidwhogg** [comment]
One issue on dates -- let's avoid the total solar eclipse by several days at least.

**Comment by jakevdp** [comment]
The eclipse is August 21st I believe – Is the 28th suitably far away? We could try for a week later.

**Comment by jakevdp** [comment]
Regarding budget: there is strong support for the Hack Week model within eScience. I'll see if I can get a firm number on what dollar amount we can count on.

**Comment by davidwhogg** [comment]
ps. IIRC, which I might not, we have had the hosting institution pay each time, which has been M-S DSE money the last two times, but we have used the individual-institution grants, not shared grants. If, however, this is too much of a burden for UW, we should open that up for discussion.

**Comment by jakevdp** [comment]
I think in 2015 eScience covered at least the cost of the space and some of the food/coffee, but we were really running on a shoestring that year

**Comment by jakevdp** [comment]
I'm going to go ahead and commit us to the week of August 28th. We need to finalize it to make sure we don't lose the space!

**Comment by drphilmarshall** [comment]
Thanks for taking care of this, Jake!
